### ISSUE TITLE
Inhibit wheel gestures on mouse gestures, and vice-versa

### ISSUE THREAD

**Issue by Reylak** [Robbendebiene/Gesturefy#270]
It's best to test this with gesture trace, display action and display gesture directions enabled.

## Wheel gesture on mouse gesture

Try this:
 
 * enable wheel gestures on the right button, with a pop up action
 * begin drawing a gesture with the right  button
 * before releasing, move the mouse wheel to trigger the pop up action
 * release the right button

Now **your mouse gesture is not triggered**, and you are stuck with a gesture trace if you have that enabled. I suggest inhibiting mouse wheel gestures during mouse gestures.

## Mouse gesture on wheel gesture

Similarly, try this:

 * enable wheel gestures on the right button, with a pop up action
 * hold right button and scroll to trigger the pop up action
 * still holding the right button, move the mouse: you begin a mouse gesture **that is triggered when releasing the right button** (with gesture trace visible)

Bonus point: do the same and start moving the mouse in the pop up; no gesture is performed until you reach any side of the pop up, at this point the trace appears between the original trigger point and your current position, registering the whole movement over the pop up as a valid, triggering direction.

I suggest inhibiting mouse gestures during wheel gesture until the trigger button is released. It's okay to leave the pop up as is, because they are closed as soon as any mouse button is pressed.

Tl;Dr: wheel gestures combined with mouse gestures can be messy, inhibit them during each other.

**Comment by Robbendebiene** [comment]
Thanks for the detailed explanation.
If I understood everything correctly this issue: https://github.com/Robbendebiene/Gesturefy/issues/123 already adresses most of these problems. Currently it's just not a the top of my todo list, since this problem should occur only rarely.

**Comment by Reylak** [comment]
Ah, yes it does, my bad.

It is more of a cosmetic mishap so as you say it's not top priority.

**Comment by Robbendebiene** [comment]
Even though I really like your issue, because it's detailed and well formated I'm closing it as a duplicate of the other one I mentioned. It will be referenced there so it wont be lost. The reason is that I'm currently facing with too many issues and therefore losing track of them. I hope you don't mind.

**Comment by Reylak** [comment]
Of course I don't mind. You're the project leader here :) And the other one came first so mine **is** a duplicate, although expanding on the problem.

**Comment by Robbendebiene** [comment]
Duplicate of #123
